Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -2922,6 +2922,34 @@ bool AccessibilityNodeObject::canSetSelectedAttribute() const
     }
 }
 
+bool AccessibilityNodeObject::isAccessibilityList() const
+{
+    RefPtr element = this->element();
+    return element ? AXListHelpers::isAccessibilityList(*element) : false;
+}
+
+bool AccessibilityNodeObject::isUnorderedList() const
+{
+    if (ariaRoleAttribute() == AccessibilityRole::List)
+        return true;
+
+    auto elementName = this->elementName();
+    return elementName == ElementName::HTML_menu || elementName == ElementName::HTML_ul;
+}
+
+bool AccessibilityNodeObject::isOrderedList() const
+{
+    if (ariaRoleAttribute() == AccessibilityRole::Directory)
+        return true;
+
+    return elementName() == ElementName::HTML_ol;
```

Comment:
Could this be written as:

`return ariaRoleAttribute() == AccessibilityRole::Directory || ariaRoleAttribute() == AccessibilityRole::Directory`